apps/judicial-system/web/src/types/index.ts

```diff
@@ -190,3 +190,8 @@ export interface TempCase
   childCase?: TempCase | null
   indictmentCounts?: TempIndictmentCount[] | null
 }
+
+export interface CourtDate {
```

Comment:
The is a type DateLog in the generated schema file that can be used instead of this interface. Please consider doing so and remove this type.